docs/content/bounties/index.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: Flow Bug Bounty Program
+description: Earn up to 100k by finding Flow bugs!
```

Comment:
"Earn 100k or more by finding ...."

Reward for critical is minimum 100k

Reply:
"Earn $100k USD value by finding Flow bugs" would be better too b/c can get rewards in different currencies.